README.md

```diff
@@ -1,2 +1,34 @@
-# examples
-Function examples for open-runtimes
+# Open Runtimes Functions Examples ⚡️
+
+![open-runtimes-box-bg-cover](https://user-images.githubusercontent.com/1297371/151676246-0e18f694-dfd7-4bab-b64b-f590fec76ef1.png)
+
+[![Discord](https://img.shields.io/discord/937092945713172480?label=discord&style=flat-square)](https://discord.gg/fP6W2qEzfQ)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+
+Example Functions to show off what you can achieve with Open Runtimes.
+
+## List of Examples
+
+| Example                     | Dart                        | Deno | Node | PHP | Python | Ruby |
+|-----------------------------|-----------------------------|------|------|-----|--------|------|
+| Welcome Email With Appwrite | [✅](/dart/welcome-email) |      |      |     |        |      |
+
+
+
+## Contributing
+
+All code contributions - including those of people having commit access - must go through a pull request and be approved by a core developer before being merged. This is to ensure a proper review of all the code.
```

Comment:
We should mention a function should have exact same functionality in all runtimes. Also, we should mention to NOT use library if possible, to keep function simple and secure.